src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -1,754 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
> This is also why the tests fail.

Never noticed this ordering causing tests to fail in Castle Core or Fortress. 

The code highlighting thing is interesting. I just thought that was VS getting confused over multi-target projects. If this is the fix for it, will happily make the change. Thanks.